data/json/items/tool/cooking.json

```diff
@@ -1030,6 +1030,22 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "water_filter_sand",
+    "type": "TOOL",
+    "name": { "str": "slow-sand water filter" },
+    "//": "Slow sand biofilter: https://wwwnc.cdc.gov/travel/yellowbook/2024/preparing/water-disinfection#:~:text=Figure%202%2D02.%20Emergency%20gravel%20and%20sand%20filter",
+    "description": "An improvised water filter made from a plastic bucket filled with a layer of sand and a layer of gravel.  It can be used to remove particulates and some pathogens from water, but it is not 100% effective.",
+    "//2": "Density of sand is around 1.6g/cm^3. Gravel is similar. pi * 5in * 5in * (10 in + 4 in) * 1.6 g/cm^3 ~= 29 kg. Add 1 kg for the bucket itself.",
+    "weight": "30 kg",
```

Comment:
```suggestion
    "weight": "14.8 kg",
```

Reply:
`CHECK( item_density( target ) <= max_density )` is failed as its density is set too high (1.42857 vs 1.3).
0.81(bucket_5gal) + 10(material_sand) + 4(material_gravel) + 0.02(wire_mesh) = 14.83 KG (now density is 0.7)